framework/src/functions/MooseParsedFunction.C

```diff
@@ -5,7 +5,7 @@
 /*           (c) 2010 Battelle Energy Alliance, LLC             */
 /*                   ALL RIGHTS RESERVED                        */
 /*                                                              */
-/*          Prepared by Battelle Energy Alliance, LLC           */
+/*          Prepared by Battelle Energy Alliadnce, LLC           */
```

Comment:
Ugh, I was testing the spell check after you asked me about spellchecking in code comments :-/